ckan_cloud_operator/providers/infra/manager.py

```diff
@@ -0,0 +1,81 @@
+import base64
+import subprocess
+import time
+import sys
+
+from ckan_cloud_operator import logs
+from ckan_cloud_operator import kubectl
+
+def get_solr_pods(format=''):
+    kubectl.call(f'get pods', 'ckan-cloud')
+
+def get_container_logs(**kubectl_args):
+    zk = kubectl_args.pop('zookeper_only', None)
+    sc = kubectl_args.pop('solrcloud_only', None)
+    suf = 'zk' if zk else 'sc'
+    service = f'ckan-cloud-provider-solr-solrcloud-{suf}-headless'
+    stream_logs = '-f ' if kubectl_args.pop('follow', None) else ''
+    k_args = [f'--{k}={v}' for k,v in kubectl_args.items() if v is not None]
+    full_args = stream_logs + ' '.join(k_args)
+    _stream_logs(f'kubectl -n ckan-cloud logs service/{service} {full_args}')
```

Comment:
From the description one would expect a default call showing all records and then use `zookeper_only` or `solrcloud_only` to restrict. When reading the logic it implements `solrcloud_only` by default or  `zookeper_only` if the user adds a parameter. Zookeeper **and** Solrcloud logs are never gonna be shown together. 

Suggestion:
 - Refactor the command to show `solrcloud` logs by default (98% of the times we check this)
 - Add a simple flag to show `zookeeper` logs if needed.
 
This will simplify the logic and the CLI API by keeping the same functionality.